repository/Cormas-Core/CormasModel.class.st

```diff
@@ -2283,6 +2283,35 @@ Example: self createFragmentedEntity: UrbanArea fromCollection: (self theCells s
 			aCollecOfSpatialEntityElement
 ]
 
+{ #category : #'+ instance creation - spatial grid' }
+CormasModel >> createGridColumns: numColumns lines: numLines neighbourhood: aNumber closed: aBoolean [
+	"Purpose: Create a spatial grid made of cells
+Arguments:  X : number of columns ; Y : number of lines of the grid.
```

Comment:
Comment should be modified in order to use the new API (no more X and Y).

Reply:
thanks for finding it out.